sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -1025,6 +1027,11 @@ case class TimeAdd(start: Expression, interval: Expression, timeZoneId: Option[S
 case class FromUTCTimestamp(left: Expression, right: Expression)
   extends BinaryExpression with ImplicitCastInputTypes {
 
+  if (!SQLConf.get.utcTimestampFuncEnabled) {
+    throw new AnalysisException(s"The $prettyName function has been disabled since Spark 3.0." +
```

Comment:
Ur, @MaxGekk . Usually, deprecation means showing warnings instead of `Exception`. This looks like a ban to me.

Reply:
Yea, deprecation is deprecation. I think we can simply deprecate them in `functions.scala`, and that's all.